res/TensorFlowLiteRecipes/Reshape_004/test.recipe

```diff
@@ -0,0 +1,31 @@
+# Test model of Reshape with 0 in shape.
+# May not work in interpreter.
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 3 dim: 2 dim: 3 }
+}
+operand {
+  name: "shape"
+  type: INT32
+  shape { dim: 3 }
+  filler { tag: "explicit" arg: "0" arg: "3" arg: "-1" }
```

Comment:
Question,
Is `0` value always converted to `1`?

Reply:
This situation is from ONNX, where dim 0 will follow that from input, not to `1`.
As of now, I've only seen model with input with dim value of `1`.

Reply:
Thanks!